schema/Cite.schema.yaml

```diff
@@ -37,6 +37,12 @@ properties:
       LaTeX's `natbib` package has `\citep{}` (for `parenthetical`), `\citet{}` (for `textual`),
       `\citeauthor{}` (for `textual-author`), `\citeyear{}` (for `textual-year`).
       See https://www.overleaf.com/learn/latex/Natbib_citation_styles.
+  citationType:
```

Comment:
Just a small suggestion, but what do you think about naming this `citationIntent` to clarify the purpose?
Personally I find nuance between `citationMode` and `citationType` harder to reason about.

Reply:
I think that is a great suggestion!